livekit-plugins/livekit-plugins-google/livekit/plugins/google/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.5.0"
+__version__ = "0.6.dev0"
```

Comment:
I don't think we should bump unchanged packages

Reply:
I think we may want to start using >= and <= in our setup.py files instead of ~